jade/hpc/hpc_submitter.py

```diff
@@ -341,6 +390,9 @@ def get_blocking_jobs(self):
     def remove_blocking_job(self, name):
         assert False
 
+    def set_blocking_jobs(self, jobs):
+        assert False
+
```

Comment:
I may missed to capture `assert False` before, `return False`, or just `pass`?

Reply:
Based on the current design, if this was ever called, it would be a bug. If it could be called, your point is valid. I put the assert there to ensure my expectations were met.